src/nos/core/notifications.py

```diff
@@ -126,3 +133,58 @@ def formatted(self, **kwargs) -> "Notification":
     "File upload completed",
     "The file {file_id} has been successfully uploaded.",
 )
+
+ALL_IVAS_INVALIDATED_TO_USER = Notification(
+    "IVA Invalidation",
+    """
+All of your IVAs have been successfully invalidated.
```

Comment:
I suggest something like "All of your contact addresses need re-verification, since you have established a new 2nd factor for authentication."

Reply:
For this and subsequent comments, see the changes in [f36fc1a](https://github.com/ghga-de/notification-orchestration-service/pull/5/commits/f36fc1a827ee69fb733e66436181234f1e838234) and let me know if that works.